packages/messages/src/useCases/chat.js

```diff
@@ -0,0 +1,30 @@
+const Logger = require('../utils/logger');
+
+// eslint-disable-next-line func-names
+const createChat = (model) => async function ({
+  hostId, customerId, messageId, place,
+}) {
+  if (hostId || customerId || messageId || place) {
```

Comment:
I think it will be better to move this into its own functionality, maybe consider using a Mongoose Validation tool https://mongoosejs.com/docs/validation.html#validation